chroma-common/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../include/Makefile.version
```

Comment:
@brianjmurrell this patch has been around on gerrit for a number of months and there are 2 follow-on patches that have similarly been around for a while, probably best to check you are not duplicating them with similar work.